docs/docs/advanced/services.mdx

```diff
@@ -1,30 +1,25 @@
 # Services
 
-LiveCodes (being a client-side app) uses multiple services (for example for authentication, sharing, module resolution, etc).
+LiveCodes (being a [client-side](../why.mdx#client-side) app) uses multiple services (for example for short-URL [sharing](../features/share.mdx), [broadcast server](../features/broadcast.mdx), authentication, module resolution, etc).
 
-These services are [defined here](https://github.com/live-codes/livecodes/tree/develop/src/livecodes/services).
-
-Some of the services are not supported on [self-hosted](../features/self-hosting.mdx) deploys and are either replaced by other compatible services or require you to provide an alternative service.
+Some of these services are not supported on _static_ [self-hosted](../features/self-hosting.mdx) deploys and are either replaced by other compatible services or require you to provide alternative ones.
 
 Examples:
 
 - The [share](../features/share.mdx) service in [self-hosted](../features/self-hosting.mdx) apps uses [dpaste](https://dpaste.com/) for short URLs, which are [**deleted after 365 days**](https://dpaste.com/help).
 - [Firebase configuration](https://github.com/live-codes/livecodes/tree/develop/src/livecodes/services/firebase.ts) for authentication.
 
-:::info
-
-LiveCodes [sponsors](../sponsor.mdx) (Bronze sponsors and above) get access to managed custom services.
+Demo for a static self-hosted app (on GitHub Pages): https://live-codes.github.io/livecodes
 
-:::
+Most likely, you will not need to provide these services yourself, and the app will work just fine using the other compatible services (e.g. dpaste).
+However, if you do need to provide them, you may wish to use the included [docker setup](../advanced/docker.mdx) which provides a self-hosted implementation for these services, very similar to the official [hosted app](https://livecodes.io).
+You can find the list of provided services [here](../advanced/docker.mdx#services).
 
-If you [self-host](../features/self-hosting.mdx) your app and need to change any of these services, start by following the [guide described there](../features/self-hosting.mdx).
+Demo for a self-hosted app that uses the [docker setup](../advanced/docker.mdx) (on a VPS): https://vps.livecodes.io
 
-Then, edit the services you want (in [`src/livecodes/services`](https://github.com/live-codes/livecodes/tree/develop/src/livecodes/services)). The used services need to have the same interface.
+:::info
```

Comment:
nit
In GitHub Flavored Markdown (GFM), :::info is not a standard syntax.

Reply:
Yes, that's not standard markdown, but it is used by Docusaurus (SSG I use for docs)

This is how it is rendered:
https://docker.livecodes.pages.dev/docs/advanced/services

https://docusaurus.io/docs/markdown-features/admonitions

Reply:
TIL!
Thanks :)